renku/core/session/docker.py

```diff
@@ -158,6 +159,19 @@ def session_start(
                         docker.types.DeviceRequest(count=[gpu_request], capabilities=[["compute", "utility"]])
                     ]
 
+            # NOTE: set git user
+            environment = {}
+            volumes = [f"{str(client.path.resolve())}:/home/jovyan/work"]
```

Comment:
careful here, the user isn't always `jovyan` (e.g. in RStudio images). Is it possible to do this after the container is up and running? Or inspect the image metadata for `USER`? It might be worth checking how vscode does this configuration when connecting a local editor to a container; they do all of it after the container is up and running afaict.